RPIPs/rpip-remove_retro_cap.md

```diff
@@ -124,13 +131,14 @@ The administrator will follow an internal handbook that they manage for their pr
 
 ##### Application Review and Interview
 
-- The Grants and Management Committee (GMC) will review the received applications within 14 days of the application window closing.
+- The Grants and Management Committee (GMC) will review the received applications once the application window closes.
 - The GMC will conduct interviews with shortlisted candidates to assess their suitability for the role.
   - The interviews may include a combination of virtual or in-person meetings, skill assessments, and reference checks.
 
 ##### Decision-Making
 
-- After the interviews and evaluations are complete, the GMC will deliberate and make a decision regarding the candidate selection.
+- After the interviews and evaluations are complete, the GMC will deliberate and make a decision regarding the candidate selection. The decision will be made through a single choice vote, where each member of the GMC selects their preferred candidate from the pool of applicants. The candidate who receives more than 50% of the votes cast by the GMC members will be selected. In the event that nobody meets the threshold (no candidate receives more than 50% of the votes), or if multiple candidates receive an equal number of votes, The GMC has 7 days to engage in further discussions to reevaluate the candidates, review their qualifications, and reconsider their merits. 
```

Comment:
Better, but... what happens after those 7 days? Another majority vote? Is it any number of majority votes can happen? And what if nobody gets selected at the end?